lib/src/konsistTest/kotlin/com/lemonappdev/konsist/api/ApiKonsistTest.kt

```diff
@@ -43,7 +52,183 @@ class ApiKonsistTest {
             }
     }
 
+    @Test
+    fun `every provider has correct hasX methods`() {
+        Konsist
+            .scopeFromPackage("com.lemonappdev.konsist.api.provider..", sourceSetName = "main")
+            .interfaces()
+            .withoutName(
+                "KoKDocTagsProvider",
+                "KoParentInterfaceProvider",
+            ) // remove after merge https://lemonappdev.atlassian.net/browse/KON-456 and release v.1.0.0
```

Comment:
Please create `1.0.0` epic and create a ticket (you can add more info on what exactly needs to be removed, so we will have better context when we grab this ticket)